Makefile

```
@@ -9,10 +9,13 @@ update:; forge update
 
 # Deployment helpers
 deploy:
-	FOUNDRY_PROFILE=production forge script script/DeployProduction.s.sol --skip src --skip test --rpc-url mainnet --slow --broadcast -vvv --verify
+	FOUNDRY_PROFILE=production MAINNET_RPC_URL=$(DEPLOY_RPC_URL) forge script script/Deploy.s.sol --skip src --skip test --slow --broadcast -vvv --verify --show-standard-json-input
```

I think you need to keep `--rpc-url mainnet`, otherwise it won't know which RPC url to use and this is defined in `foundry.toml`.
https://github.com/m0-foundation/wrapped-m-token/blob/448d79c33f066ffc00373bc2238c37cedde1d1bf/foundry.toml?plain=1#L34

```suggestion
	FOUNDRY_PROFILE=production MAINNET_RPC_URL=$(DEPLOY_RPC_URL) forge script script/Deploy.s.sol --skip src --skip test --rpc-url mainnet --slow --broadcast -vvv --verify --show-standard-json-input
```

---

So I removed the `MAINNET_RPC_URL=$(DEPLOY_RPC_URL)` part and added in `--rpc-url ${DEPLOY_RPC_URL} --etherscan-api-key ${ETHERSCAN_API_KEY}` which will come from the `.env` file.